specification/hybridcompute/resource-manager/Microsoft.HybridCompute/preview/2023-03-15-preview/HybridCompute.json

```diff
@@ -0,0 +1,2197 @@
+        },
+        "subnet": {
+          "type": "object",
+          "$ref": "#/definitions/Subnet",
+          "description": "The subnet to which this IP address belongs."
+        }
+      },
+      "description": "Describes properties of the IP address."
+    },
+    "Subnet": {
+      "type": "object",
+      "readOnly": true,
+      "properties": {
+        "addressPrefix": {
+          "type": "string",
+          "description": "Represents address prefix."
+        }
+      },
+      "description": "Describes the subnet."
+    },
+    "HybridIdentityMetadataList": {
+      "description": "List of HybridIdentityMetadata.",
+      "type": "object",
+      "properties": {
+        "nextLink": {
+          "description": "Url to follow for getting next page of HybridIdentityMetadata.",
+          "type": "string"
+        },
+        "value": {
+          "description": "Array of HybridIdentityMetadata",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/HybridIdentityMetadata"
+          }
+        }
+      },
+      "required": [
+        "value"
+      ]
+    },
+    "HybridIdentityMetadata": {
+      "description": "Defines the HybridIdentityMetadata.",
+      "required": [
+        "properties"
+      ],
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ProxyResource",
+          "description": "The resource model definition for an Azure Resource Manager proxy resource. It will have everything other than required location and tags."
+        }
+      ],
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "description": "Resource properties.",
+          "$ref": "#/definitions/HybridIdentityMetadataProperties"
+        },
+        "systemData": {
+          "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/systemData",
+          "description": "The system data."
+        }
+      },
+      "x-ms-azure-resource": true
+    },
+    "HybridIdentityMetadataProperties": {
+      "description": "Defines the resource properties.",
+      "type": "object",
+      "properties": {
+        "vmId": {
+          "description": "The unique identifier for the resource.",
+          "type": "string"
+        },
+        "publicKey": {
+          "description": "The Public Key.",
+          "type": "string"
+        },
+        "identity": {
+          "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/Identity",
+          "readOnly": true
+        }
+      }
+    },
+    "AgentVersionsList": {
+      "type": "object",
+      "readOnly": true,
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/AgentVersion"
+          },
+          "description": "The list of available Agent Versions.",
+          "x-ms-identifiers": []
```

Comment:
>] [](http://example.com/codeflow?start=31&length=1)

specify the property that makes each element unique.

Reply:
agentVersion in the AgentVersion object will be the unique property.
Is there something we need to add in x-ms-identifiers? If so, can you provide some examples?

Reply:
Hi Roopesh, according to [https://azure.github.io/autorest/extensions/#x-ms-identifiers](https://azure.github.io/autorest/extensions/#x-ms-identifiers) guidelines, "If there is no appropriate identifying properties, leave it as an empty array". I think we are fine in this case.

Reply:
Added, thanks!